packages/mint-components/src/components/sqm-referral-table/columns/referral-table-rewards-column.feature

```diff
@@ -2,62 +2,71 @@
 @owner:noah
 Feature: Referral Table Reward Column
 
-    Shows the reward associated with each referral
+  Shows the reward associated with each referral
 
-    Background:
-        Given the status column is included in the referral table
-        And at least one referral exists
+  Background:
+    Given the status column is included in the referral table
+    And at least one referral exists
 
-    @motivating
-    @ui
-    Scenario Outline: The referral reward and it's status are shown for each referral
-        Then for each referral reward there exists a reward cell
-        And the reward type and value is displayed in the cell
-        And the status of each reward is displayed as a pill in the cell
-        And rewards of <status> have a <pillColour> pill with the text <statusText>
-        Examples:
-            | status         | pillColour | statusText     |
-            | Available      | green      | Available      |
-            | Pending        | orange     | Pending        |
-            | Cancelled      | red        | Cancelled      |
-            | Expired        | red        | Expired        |
-            | Redeemed       | blue       | Redeemed       |
-            | Pending Review | orange     | Pending Review |
-            | Denied         | red        | Denied         |
+  @motivating
+  @ui
+  Scenario Outline: The referral reward and it's status are shown for each referral
+    Then for each referral reward there exists a reward cell
+    And the reward type and value is displayed in the cell
+    And the status of each reward is displayed as a pill in the cell
+    And rewards of <status> have a <pillColour> pill with the text <statusText>
+    Examples:
+      | status                   | pillColour | statusText     |
+      | Available                | green      | Available      |
+      | Pending                  | orange     | Pending        |
+      | Pending Tax Review       | orange     | Pending        |
+      | Pending New Tax Form     | orange     | Pending        |
+      | Pending Tax Submission   | orange     | Pending        |
+      | Pending Partner Creation | orange     | Pending        |
+      | Cancelled                | red        | Cancelled      |
+      | Payout Failed            | red        | Payout Failed  |
+      | Expired                  | red        | Expired        |
+      | Redeemed                 | blue       | Redeemed       |
+      | Payout Sent              | blue       | Payout Sent    |
+      | Pending Review           | orange     | Pending Review |
+      | Denied                   | red        | Denied         |
```

Comment:
Even though the pending case is detailed in subsequent scenarios it's probably worth mentioning here so the table is "complete"